src/js/App/Sidenav/Navigation/DynamicNav.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import { useModule } from '@scalprum/react-core';
+import PropTypes from 'prop-types';
+import { Skeleton } from '@patternfly/react-core';
+import ChromeNavItemFactory from './ChromeNavItemFactory';
+import { useHistory } from 'react-router';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadLeftNavSegment } from '../../../redux/actions';
+
+const DynamicNav = ({ dynamicNav, ...rest }) => {
+  const [nav, setNav] = useState();
+  const {
+    location: { pathname },
+  } = useHistory();
+  const dispatch = useDispatch();
+  const [appName] = dynamicNav.split('/');
+  const currentNamespace = pathname.split('/')[1];
```

Comment:
I think we have a helper function for this somewhere. And if not, the same split is definitely done somewhere in the nav already. Might be worth extracting it.

Reply:
It's used on couple of places, but not direct export. I'll create one in `utils` file.